crates/vm/levm/src/vm.rs

```diff
@@ -577,6 +617,8 @@ impl VM {
             }
         }
 
+        // Access List validations ?
```

Comment:
Are there any validations that should be implemented? AFAIK there are no validations that should prevent execution regarding access lists. I only know the gas related validations (that are already checked).
Let me know if you found that we have to implement more validations.

Reply:
I left the comment in case I found any, but I could not find any, so I should erase the comment.

Reply:
Deleted in [Delete comment in prepare_execution](https://github.com/lambdaclass/ethrex/pull/1447/commits/a6605e1007d15d8caefdf43e96137a2aac4002f6)